components/DetailPageSidebar/detail-page-sidebar.scss

```diff
@@ -0,0 +1,131 @@
+@import '~carbon-components/scss/globals/scss/styles.scss';
```

Comment:
This component should only import the styles it needs, not the entirity of the Carbon scss